lib/account.php

```diff
@@ -123,6 +123,7 @@ public function createMailbox($mailBox) {
 		$conn->createMailbox($mailBox);
 		$this->mailboxes = null;
 
+//		$mailBox = \Horde_Imap_Client_Mailbox::get($mailBox, false)->utf7imap;
```

Comment:
Ohh this is a new line, not commenting out old lines..
We should remove this as it makes no sense, I think.